networkx/algorithms/community/tests/test_local.py

```diff
@@ -0,0 +1,67 @@
+import networkx as nx
+
+
+def test_clauset_karate_club():
```

Comment:
Why is it called karate club?

Reply:
I used this name as it is testing the algorithm against the standard karate club graph. Happy to rename if there is a more appropriate name?